examples/pizza-order/README.md

```diff
@@ -0,0 +1,153 @@
+# Pizza order example
+
+*This example demonstrates how to use miranum connect and how you can decouple microservices using it.*
+
+There are four services in total:
+
+* `waiter`: responsible for placing the order, serving beverages and issuing the check
+* `kitchen`: responsible for the cooking
+* `email`: responsible for notifying the customer for the order confirmation
+* `frontend`: frontend to place order to the waiter service by REST
+
+## Run the project
+
+1. Start all services, namely the `WaiterC8Application`, `KitchenC8Application`, `EmailC8Application` and the `PizzaOrderFrontEnd`
+2. Make sure that you have a running instance of the Camunda8 Engine running. <br> *If not, just start the camunda-8 stack under `stack/camunda-8`.*
+3. Also make sure to deploy the `pizza-order-c8.bpmn` to the running Camunda8-Engine.
+4. Now navigate to http://localhost:5050 to view the Frontend-Application. You will end up seeing something like this:
+
+<br>
+<div align="center">
+    <img src="./../../images/pizza-order-frontend.png" alt="pizza-order-frontend" width="300" align="center">
+</div>
+<br>
```

Comment:
Resizing an image depends on the markdown engine used in the background (e.g. README.md on different website may look different). Using html in markdown is a common thing I would not start changing that at every place in the project. How would you otherwise change the size of an image?

Reply:
Imho the Markdown language is intended primarily to simplify writing without adding the complexity of HTML. Hence, I would not include HTML tags for sizing images. 

At Camunda it was considered as a bad practice and not allowed in the documentation. It can lead to more and more unwanted complexity in a readme file which should be rather simple. (e.g. by adding a .css file to make it look even better etc.) To me that is not the intent of such readme. 
But as said, that's a discussion point I wanted to raise here. 

I would simply use: 
`![Description](path/to/pic)`
And make sure that the pictures itself are sized properly. (For instance: Rectangular diagrams around 500x1200px, and square diagrams around 500x500px)